ttnn/cpp/ttnn/operations/conv/conv2d/conv2d_utils.cpp

```diff
@@ -162,6 +127,9 @@ ParallelConfig determine_parallel_config(
                 ? find_closest_largest_divisor_with_num_padding(
                       out_channels_ntiles, input_channles_ntiles, start_divisor_c)
                 : find_closest_largest_divisor(out_channels_ntiles, input_channles_ntiles, start_divisor_c);
+        if(is_non_tile_mul_shard_width) {
+            num_cores_c = find_closest_largest_divisor(out_channels_ntiles, input_channles_ntiles, start_divisor_c);
```

Comment:
any reason why we are not using find_closest_largest_divisor_with_num_padding here?
It should provide better parallelisation?